framework/packages/abstract-client/src/account.rs

```diff
@@ -221,15 +268,25 @@ impl<Chain: CwEnv> Account<Chain> {
             .find(|a| a.key == "sub_account_added")
             .map(|a| a.value.parse().unwrap());
 
-        let module_address: Option<String> = wasm_abstract_attributes
+        let module_addresses: Option<String> = wasm_abstract_attributes
             .iter()
             .find(|a| a.key == "new_modules")
             .map(|a| a.value.parse().unwrap());
 
+        // When there are multiple modules registered the addresses are returned in a common
```

Comment:
This comment is not entirely correct because the instantiation logic actually doesn't assert any order in the install. However, if you can ensure that the dependency logic always puts the top-level module last, then this is correct. But the comment should probably reflect that nuance

Reply:
Yes good idea